test/extensions/filters/network/http_connection_manager/config_test_base.h

```diff
@@ -46,6 +50,21 @@ class HttpConnectionManagerConfigTest : public testing::Test {
   }
 };
 
+class PassThroughFilterFactory : public Extensions::HttpFilters::Common::FactoryBase<
```

Comment:
is this needed?

Reply:
I think it's useful, I expect it to be used elsewhere and would probably try to colocate it with PassThroughFilter if that becomes the case. Right now it's extended by ChefFilterFactory and ChefFilterFactory.